experimental/cloudstrapper/playbooks/roles/build-platform/vars/main.yaml

```diff
@@ -0,0 +1,59 @@
+---
+
```

Comment:
How do we plan to keep this in sync as people continue making changes in Magma ? Should we have some script to generate this ?

Reply:
IMHO, this file needs to change whenever there is a change in our build environment. If we choose to use Cloudstrapper as the primary install vehicle and make changes to the globally available documentation, this process would become a release gate.